src/Datadog.Trace.ClrProfiler.Managed/Integrations/GraphQLIntegration.cs

```diff
@@ -307,9 +307,9 @@ private static void RecordExecutionErrorsIfPresent(Span span, string errorType,
 
             if (errorCount > 0)
             {
-                span.Error = true;
-
-                span.SetTag(Trace.Tags.ErrorMsg, $"{errorCount} error(s)");
+                var message = $"{errorCount} error(s)";
+                span.Status = SpanStatus.Error.WithDescription(message);
+                span.SetTag(Trace.Tags.ErrorMsg, message);
                 span.SetTag(Trace.Tags.ErrorType, errorType);
                 span.SetTag(Trace.Tags.ErrorStack, ConstructErrorMessage(executionErrors));
```

Comment:
We'll probably need to handle these using the semantic conventions abstraction, because they are probably required for Datadog, but they are not part of the OTel semantic conventions. Either way, that work can be done separately from this PR.

Reply:
I will mention this comment under https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/85